docusaurus/docs/reactnative/guides/video_attachment.mdx

```diff
@@ -0,0 +1,141 @@
+---
+id: video-attachment
+sidebar_position: 6
+title: Video Attachment
+---
+
+We would be building a custom video component in the application using some of the available open-source packages and using the `FileAttachment` customization prop which the `Channel` component allows us to use.
+
+Uploading a video attachment is possible using the file upload in the attachment picker.
+
+To add the video attachment and view the file we follow these steps:
+
+## Installing the packages
+
+We use two packages for rendering the video and creating the thumbnail of the video.
+For rendering the video in the message list we recommend using the [react-native-video](https://www.npmjs.com/package/react-native-video) package.
+To create the thumbnail of the video attachment we recommend using the [react-native-create-thumbnail](https://github.com/souvik-ghosh/react-native-create-thumbnail) package.
+
+Note: [react-native-video](https://www.npmjs.com/package/react-native-video) and [react-native-create-thumbnail](https://github.com/souvik-ghosh/react-native-create-thumbnail) are external package.
+
+After installing it do not forget to link the package to your project. Follow the guide in the documentation for Android and iOS installation for `react-native-video` - [here](https://github.com/react-native-video/react-native-video#installation) and for `react-native-create-thumbnail` - [here](https://github.com/souvik-ghosh/react-native-create-thumbnail#getting-started).
+
+After the package is installed, we can use the `Video` component it provides to render video.
+
+## Creating a Video Card Thumbnail component
+
+To create a video thumbnail we have already installed the package `react-native-create-thumbnail`. This package takes in the attachment `asset_url` and returns the thumbnail image URL after saving it to the local file directory. You can also cache the file so that it doesn't gets saved every time the screen is loaded.
```

Comment:
```suggestion
To create a video thumbnail we have already installed the package `react-native-create-thumbnail`. This package takes in the attachment `asset_url` and returns the thumbnail image URL after saving it to the local file directory. You can also cache the file so that it doesn't get saved every time the screen is loaded.
```